backend/src/api/groups/groups.api.ts

```diff
@@ -27,6 +30,10 @@ type Options = {
 const initGroupsApi: FastifyPluginAsync<Options> = async (fastify, opts) => {
   const { group: groupService } = opts.services;
 
+  fastify.addHook(ControllerHook.PRE_HANDLER, async (request) => {
+    await checkHasPermissions(PermissionKey.MANAGE_UAM)(request);
+  });
```

Comment:
```suggestion
```
you need to add it directly on the route
<img width="513" alt="image" src="https://user-images.githubusercontent.com/42296182/184613217-ca4a17e5-d222-4dbf-b58d-f8956e4d4945.png">

Reply:
But then I have to put in on every of 3 routes

Reply:
I don't understand what's the purpose of doing it this way